html-aam/accname-computation-by-element/input-image.html

```diff
@@ -0,0 +1,59 @@
+<!doctype html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <title>Accname computation tests for element: &lt;input type="image"&gt;</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/resources/testdriver.js"></script>
+  <script src="/resources/testdriver-vendor.js"></script>
+  <script src="/resources/testdriver-actions.js"></script>
+  <script src="/wai-aria/scripts/aria-utils.js"></script>
+  <script src="/html-aam/scripts/html-aam-accname-utils.js"></script>
+</head>
+
+<body>
+  <script>
+    const testConfig = {
+      urlSpec: 'https://www.w3.org/TR/html-aam-1.0/#input-type-image-accessible-name-computation',
+      elements: {
+        'input[type="image"]': { attrs: ['img src']}
+      },
+      nameSources: [
+        'from 2 aria-labelledby refs',
+        'from 1 aria-labelledby ref',
+        'from aria-label',
+        'from 2 labels',
+        'from 1 label',
+        'from alt',
+        'from empty alt',
```

Comment:
The specs don't clearly state whether alt="" should be used before title. Unlike with <img>, it's not explicitly defined how these attributes should impact if present together. In practice, most major browsers prioritize the title attribute over alt="".